### ISSUE TITLE
v0.8 / in Grant Request Authorization Page - Links should be disabled

### ISSUE THREAD

**Issue by tclaus** [diaspora/diaspora#8168]
I am testing around with new OpenID Login for a mobile client. 
When the page "Grant Request to Scopes" arises (the page to grant access to the different scopes, don't know the real name) - the headline contains the usual diaspora links. 
On this page, no links (especially not on mobiles) should be disabled or a special page should be designd.
Currently all links in the head do work and produce a mess when clicked during permission process: 

<img width="414" alt="Bildschirmfoto 2020-10-24 um 14 09 16" src="https://user-images.githubusercontent.com/501326/97081574-9080b880-1603-11eb-8b2b-e7c0aa7fd68f.png">

**Comment by jhass** [comment]
Maybe we should just hide the header there entirely, that would be pretty easy to implement by just changing the layout for the view in https://github.com/diaspora/diaspora/blob/7fabd9d17f29d3bbbc40e8c31c1e33a2a8875464/app/controllers/api/openid_connect/authorizations_controller.rb#L116-L125

What do others think?

**Comment by SuperTux88** [comment]
I think I'd still like the diaspora logo somewhere on that page, to make clear that this page belongs to diaspora. But it doesn't need a full header (and also doesn't need to be clickable).

**Comment by denschub** [comment]
I agree with @SuperTux88. Just a black bar with the star centered should be enough to make existing users know where they're at. Not making it clickable is probably a good idea, since landing on the index page would kinda ruin the auth flow entirely. :)
